types/greasemonkey/index.d.ts

```diff
@@ -275,6 +275,16 @@ declare var GM: {
      */
     openInTab(url: string, openInBackground?: boolean): void
 
+    /**
+     * Adds an item to the User Script Commands menu.
+     * @param caption The caption to display on the menu item.
+     * @param commandFunc The function to call when the menu item is selected.
+     * @param accessKey A single character that can be used to select the
+     * item when the menu is open. It should be a letter in the caption.
+     * @see {@link https://wiki.greasespot.net/GM.registerMenuCommand}
+     */
+    registerMenuCommand(caption: string, commandFunc: () => any, accessKey?: string): void;
```

Comment:
I think it should `commandFunc: () => void`, because the docs don't say returned values is taken into account

Reply:
Updated the signature to `() => void`